ngraph/frontend/onnx/frontend/include/onnx_frontend/input_model.hpp

```diff
@@ -0,0 +1,34 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <frontend_manager/input_model.hpp>
+#include <onnx_editor/editor.hpp>
+
+namespace ngraph
+{
+    namespace frontend
+    {
+        class FRONTEND_API InputModelONNX : public InputModel
+        {
+        public:
+            InputModelONNX(const std::string& path);
+
+
+            std::vector<Place::Ptr> get_inputs() const override;
+            Place::Ptr get_place_by_tensor_name(const std::string& tensor_name) const override;
+            void set_partial_shape(Place::Ptr place, const ngraph::PartialShape& shape) override;
```

Comment:
Is there tests for these 'set_partial_shape' methods? You can consider to reuse test suites from ngraph/test/frontend/shared

Reply:
The tests will be added in my PR (already test draft: https://github.com/openvinotoolkit/openvino/pull/6646), where editor features are exposed in FE API (but #6615 should be merged at first)